### ISSUE TITLE
stage accuracy

### ISSUE THREAD

**Issue by NirHeaven** [mpc001/end-to-end-lipreading#4]
Hello, I'm trying to train the lip reading network following by your advice,could you please tell me the accuracy that the model can achieve for three stages?

**Comment by mpc001** [comment]
Sure. Here are validation accuracy for three stages.

| Stream        | 1st stage           | 2nd stage  | 3rd stage  |
| :-------------: |:-------------:| :-----:| :-----:|
| video      | 0.7636 | 0.7852  | 0.8366  |
| audio      | 0.9572 | 0.9703  | 0.9778  |
| audiovisual |  -     | 0.9760   | 0.9836  |

**Comment by NirHeaven** [comment]
I get another problem, all the accuracy are 0!Is there something wrong in statices?First stage with loss about 1.7

**Comment by NirHeaven** [comment]
hello，第一阶段的话，loss大概到多少acc才开始不是0？这边训练到3.多，acc还是？

**Comment by NirHeaven** [comment]
找到原因了。。
import torch
a = torch.Tensor([[2,4,5,6], [3,6,7,3],[12,54,2,43]])
b = torch.Tensor([1,2,3]).long()
c = 0.
_, ind = torch.max(a, 1)
c += torch.sum(ind==b.data)

print("{:.4f}".format(c/len(a)))
这段代码结果恒为0,torch里面的sum出来的是long类型，这样会先除后转float，结果就为0了，需要手动把c强转成float

**Comment by WillQuCD** [comment]
Could you tell me how many hours each stage takes?
